src/lib/components/FloatingArrow/FloatingArrow.test.svelte.ts

```diff
@@ -17,4 +18,58 @@ describe('FloatingArrow', () => {
 			expect(component).toBeInTheDocument();
 		}),
 	);
+
+	it(
+		'renders position based on context placement',
+		withEffect(() => {
+			const arrowRef = document.createElement('div');
+			const floating = useFloating({ placement: 'left' });
+			const { getByTestId } = render(FloatingArrow, {
```

Comment:
Instead of:
```js
const { getByTestId } = render(FloatingArrow);
```
testing library exposes a `screen` object so you can actually do:
```js
render(FloatingArrow);

screen.getByTestId('...');
```
They work identically but I've used `screen` since the testing library example showcased that. I suggest we just follow that since all the other tests are also using this.
It's imported from `@testing-library/svelte`

Reply:
Any benefit to this? I often times swap between `getByTestId` to other query methods on the fly, so this makes it easier to do so as I jump from test to test.

Reply:
screen has autocomplete and doesn't require to destructure every test, less visual clutter.

Reply:
Done